README.md

```diff
@@ -30,7 +30,12 @@ There are two main categories of samples: **Camera** and **Applications**. The s
       - [**ReadIterateZDF**][ReadIterateZDF-url] - Read point cloud data from a ZDF file, iterate through it, and extract individual points.
       - [**ZDF2PLY**][ZDF2PLY-url] - Convert point cloud from ZDF a file to a PLY file.
   - **Advanced**
-    - [**HandEyeCalibration**][HandEyeCalibration-url] - Perform Hand-Eye calibration.
+    - [**HandEyeCalibration**][HandEyeCalibrationReadme-url]
+        - [**HandEyeCalibration**][HandEyeCalibration-url] - Perform Hand-Eye calibration.
```

Comment:
Can you separate out these two lines of code from this and add them to the previous commit. This is basically showing moving the sample.

Reply:
done